me-md/me-md
# Implement PR Template and set up WelcomePage component

## What does this PR do?
​This PR adds a `.github` directory with a `pull_request_template.md` file, updates the testing portion of the scripts package in `package.json`, and adds a WelcomePage component with initial structure and base information the user will see.
​
### Where should the reviewer start?
​Please look over `/Components/WelcomePage/WelcomePage.js` to review the code that was added from the pairing session today. There are two sizable files in the `assets/images` directory now, but those are simply the icons we agreed upon from `flaticon.com`.
​
#### How should this be manually tested?
​Once pulling down, please start the application locally and see if you see the WelcomePage component render! You should see a blue header (with a gradient), an icon of a female doctor, a Welcome statement and a button to `BEGIN`
​
#### Any background context you want to provide?
​We need to look into the button styling issue, perhaps go with a different component. 
​
#### What are the relevant tickets?
​Closes #20 
​
#### Screenshots (if appropriate)
<img width="535" alt="iPhone X" src="https://user-images.githubusercontent.com/31895658/71050678-1211db00-2103-11ea-8c58-edb167991563.png">
<img width="535" alt="iPhone 6S" src="https://user-images.githubusercontent.com/31895658/71050691-1b9b4300-2103-11ea-8a9c-1a88e55441f2.PNG">
​
#### Additional Notes

None
